src/views/LoginSuccess.vue

```diff
@@ -81,8 +90,14 @@ export default class LoginSuccess extends Vue {
             })),
         };
 
-        this.title = 'Your account is ready.';
-        this.state = Loader.State.SUCCESS;
+        if (this.gotError) {
+            this.state = Loader.State.WARNING;
+            this.title = `Your addresses may be
+incomplete.`;
```

Comment:
I think it's bad UX when the user gets automatically redirected after seeing a warning. I think in case of the warning, the user needs to explicitly click a "Ok, check my account" action button in the warning screen and _not_ get automatically redirected back to the Safe.

Reply:
Additionally, you should set the `status` on the `Loader` to add some more text, for example: "We might have missed used addresses that have no balance."

Reply:
But only the above text, if that was actually the error coming from the WalletInfoCollector. If another core error happened, that should be displayed.